ush/rstprod.sh

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/env bash
+
+source "$HOMEgfs/ush/preamble.sh"
```

Comment:
If we definitely need this script from the v17 package then remove the preamble line.